### PR TITLE
Update dcos node diagnostic deprecation message

### PR DESCRIPTION
**Pull request by janisz**
Fixes: COPS-6209

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-ci run Windows integration tests

**Comment by janisz** [comment]
@pierrebeitz Can you look at this?

File: pkg/cmd/node/node_diagnostics.go [hunk 1]
```diff
@@ -14,7 +14,8 @@ func newCmdNodeDiagnostics(ctx api.Context) *cobra.Command {
 		Use:   "diagnostics",
 		Short: "Use diagnostics bundles",
 		RunE: func(cmd *cobra.Command, args []string) error {
-			err := ctx.Deprecated("This command is deprecated since DC/OS 2.0, please use 'dcos diagnostics' instead.")
+			err := ctx.Deprecated("'dcos node diagnostics' command is deprecated since 2.0, but still functional in this release." +
```
Comment (pierrebeitz) [inline pkg/cmd/node/node_diagnostics.go:17]: just a really minor nitpick: i'd write `'dcos node diagnostics' is deprecated`.

**Review by pierrebeitz — APPROVED** [review]
the changes look reasonable to me.

i did not compile and test this.

File: pkg/cmd/node/node_diagnostics.go (hunk 1 above)
Comment (janisz) [reply pkg/cmd/node/node_diagnostics.go:17]: I'd stay with this just like described in ticket.

**Pull request merged**